packages/chain-adapters/src/bitcoin/BitcoinChainAdapter.test.ts

```diff
@@ -143,6 +144,24 @@ describe('BitcoinChainAdapter', () => {
     }
   })
 
+  describe('constructor', () => {
+    it('should return chainAdapter with no chainId', () => {
+      const adapter = new bitcoin.ChainAdapter(args)
+      const chainId = adapter.getChainId()
+      expect(chainId).toEqual(VALID_CHAIN_ID)
+    })
+    it('should return chainAdapter with valid chainId', () => {
+      args.chainId = VALID_CHAIN_ID
+      const adapter = new bitcoin.ChainAdapter(args)
+      const chainId = adapter.getChainId()
+      expect(chainId).toEqual(VALID_CHAIN_ID)
```

Comment:
Can you change this test to use a different ChainID value than the default one? This test would still pass even if the code completely ignored the `chainId` argument. 

Bitcoin testnet: `bip122:000000000933ea01ad0ee984209779ba`